torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8503,6 +8559,45 @@ def ref_pairwise_distance(input1, input2):
                              precisionOverride({torch.float: 1e-4, torch.cfloat: 1e-4}),
                              'TestFFT', 'test_reference_nd')],
                      ),
+    OpInfo('fft.fftshift',
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.half),
+           sample_inputs_func=lambda *a, **kw: list(sample_inputs_fftshift(*a, **kw)),
+           supports_out=False,
+           supports_forward_ad=True,
+           ),
+    OpInfo('fft.ifftshift',
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.half),
+           sample_inputs_func=lambda *a, **kw: list(sample_inputs_fftshift(*a, **kw)),
+           supports_out=False,
+           supports_forward_ad=True,
+           ),
+    OpInfo('stft',
+           decorators=[
+               skipCPUIfNoFFT,
+               DecorateInfo(unittest.skip("Skipped! stft does not match the native function"),
+                            'TestJit', 'test_variant_consistency_jit'),
+           ],
+           dtypes=floating_and_complex_types(),
+           sample_inputs_func=lambda *a, **kw: list(sample_inputs_stft(*a, **kw)),
+           check_batched_grad=False,
+           check_batched_gradgrad=False,
+           supports_out=False,
+           gradcheck_nondet_tol=GRADCHECK_NONDET_TOL,
+           ),
+    OpInfo('istft',
+           decorators=[
+               skipCPUIfNoFFT,
+               DecorateInfo(unittest.skip("Skipped! istft does not match the native function"),
+                            'TestJit', 'test_variant_consistency_jit'),
+           ],
+           dtypes=floating_and_complex_types(),
+           # FIXME: col2im does not support automatic differentiation for outputs with complex dtype.
```

Comment:
And the next PR in the stack doesn't resolve this?

Reply:
It does. That's why I remove the fixme note in the next pr.

Reply:
Right; thanks; I just missed that